### PR TITLE
Fix list of tables affecting simplified parcel layers

### LINKED ISSUE #165: LDS_MaintainSimplifiedParcelLayers runs when changes occur in unused tables
While trying to find possible UTF8 sources in LDS tables, I stumbled upon what looks like being an over-sensibility of `LDS_MaintainSimplifiedParcelLayers`, which is coded to only trigger regeneration of tables when changes occurred, for the given upload identifier, in a set of tables.

Of the set of tables, not all of them seem to be really used in the queries generating the output table, thus making such tables not worth checking for updates.

Full list of tables checked for changes:
                    'crs_affected_parcl',
                    'crs_appellation', 
                    'crs_estate_share',
                    'crs_legal_desc',
                    'crs_legal_desc_prl',
                    'crs_locality',
                    'crs_parcel',
                    'crs_proprietor',
                    'crs_stat_act_parcl',
                    'crs_statute',
                    'crs_statute_action',
                    'crs_survey',
                    'crs_sys_code',
                    'crs_title',
                    'crs_title_estate',
                    'crs_ttl_hierarchy',
                    'crs_topology_class',
                    'crs_work',
                    'cbe_title_parcel_association'

Of the above, I can not find other references, in the `04-lds_layer_functions.sql.in`, of the following tables:

                    'crs_appellation', 
                    'crs_legal_desc_prl',
                    'crs_statute',
                    'crs_statute_action',
                    'crs_ttl_hierarchy',

Shall we skip checking those 5 tables then ? \cc @palmerj

### PR DESCRIPTION
**Pull request by strk**
Closes #165

### CODE REVIEW

**Comment by imincik** [comment]
I can't find those tables in `LDS_MaintainSimplifiedParcelLayers` function but I am wondering if there is some other reason why this function is executed when they are changed.

**Comment by strk** [comment]
@palmerj please if you can review this today I'd include it in next release which would be due for this sprint

File: sql/04-lds_layer_functions.sql.in [hunk 1]
```diff
@@ -1566,21 +1566,16 @@ BEGIN
...
                     'crs_parcel',
                     'crs_proprietor',
                     'crs_stat_act_parcl',
-                    'crs_statute',
-                    'crs_statute_action',
                     'crs_survey',
                     'crs_sys_code',
                     'crs_title',
                     'crs_title_estate',
-                    'crs_ttl_hierarchy',
```
Comment (palmerj) [inline sql/04-lds_layer_functions.sql.in]: This can be deleted

File: sql/04-lds_layer_functions.sql.in [hunk 2]
```diff
@@ -1566,21 +1566,16 @@ BEGIN
                 p_upload,
                 ARRAY[
                     'crs_affected_parcl',
-                    'crs_appellation',
```
Comment (palmerj) [inline sql/04-lds_layer_functions.sql.in]: bde_get_combined_appellation needs crs_appellation

File: sql/04-lds_layer_functions.sql.in [hunk 3]
```diff
@@ -1566,21 +1566,16 @@ BEGIN
...
                     'crs_estate_share',
                     'crs_legal_desc',
-                    'crs_legal_desc_prl',
                     'crs_locality',
                     'crs_parcel',
                     'crs_proprietor',
                     'crs_stat_act_parcl',
-                    'crs_statute',
```
Comment (palmerj) [inline sql/04-lds_layer_functions.sql.in]: bde_get_par_stat_act needs both crs_statute and crs_statute_action

**Comment by strk** [comment]
@palmerj I've reintroduced, with associated comments, the tables you confirmed affected the operations. Left-over removals are:
  - crs_legal_desc_prl
  - crs_ttl_hierarchy (you confirmed this one can be removed)

**Pull request merged**